templates/frontend/components/searchForm_simple.tpl

```diff
@@ -12,7 +12,7 @@
 {if !$currentJournal || $currentJournal->getData('publishingMode') != $smarty.const.PUBLISHING_MODE_NONE}
 	{capture name="searchFormUrl"}{url page="search" op="search" escape=false}{/capture}
 	{$smarty.capture.searchFormUrl|parse_url:$smarty.const.PHP_URL_QUERY|parse_str:$formUrlParameters}
-	<form class="pkp_search" action="{$smarty.capture.searchFormUrl|strtok:"?"|escape}" method="get" role="search">
+	<form class="pkp_search" action="{$smarty.capture.searchFormUrl|strtok:"?"|escape}" method="get" role="{translate|escape key="submission.search"}">
```

Comment:
Instead of replacing the `role`, we want to add a new `aria-label` attribute. 

The `role` identifies a [landmark role](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/search.html) for assistive technology. So that needs to be set to `search`. Then the `aria-label` provides a label that assistive technology will read out to describe the area.

So instead of:

```
role="{translate|escape key="submission.search"}"
```

It should be:

```
role="search" aria-label="{translate|escape key="submission.search"}"
```

Same with OMP.

Reply:
Ok!